### PR TITLE
Fix: DL on MacOS

### PR DESCRIPTION
**Pull request by kalisp**
## Brief description
This works if DL Openpype plugin Installation Directories is set to level of app bundle (eg. '/Applications/OpenPype 3.15.0.app')

### CODE REVIEW

**Comment by kalisp** [comment]
Previous error was fixe by setting environment in System Settings:
```
'LC_ALL'='en_US.UTF-8'
'LANG'='en_US.UTF-8'
```
Not sure if is not only case of our test machine.

**Comment by antirotor** [comment]
> Previous error was fixe by setting environment in System Settings:
> 
> ```
> 'LC_ALL'='en_US.UTF-8'
> 'LANG'='en_US.UTF-8'
> ```
> 
> Not sure if is not only case of our test machine.

I guess this is related to how Python is build using pyenv

**Review by antirotor — APPROVED** [review]
have not tested it, but these changes looks good

**Comment by antirotor** [comment]
Now it's even better! :)

File: openpype/modules/deadline/repository/custom/plugins/GlobalJobPreLoad.py [hunk 1]
```diff
@@ -189,6 +189,12 @@ def get_openpype_executable():
     exe_list = config.GetConfigEntryWithDefault("OpenPypeExecutable", "")
     dir_list = config.GetConfigEntryWithDefault(
         "OpenPypeInstallationDirs", "")
+
+    # clean '\ ' for MacOS pasting
+    if exe_list:
+        exe_list = exe_list.replace("\\ ", " ")
+    if dir_list:
+        dir_list = dir_list.replace("\\ ", " ")
```
Comment (iLLiCiTiT) [inline openpype/modules/deadline/repository/custom/plugins/GlobalJobPreLoad.py:197]: If it's Mac specific it should happen on Mac. And I think the check of value is not needed as it should return always string.
```suggestion
    # clean '\ ' for MacOS pasting
    if platform.system().lower() == "darwin":
        exe_list = exe_list.replace("\\ ", " ")
        dir_list = dir_list.replace("\\ ", " ")
```

**Review by iLLiCiTiT — APPROVED** [review]

**Review by 64qam — APPROVED** [review]

**Pull request merged**